### PR TITLE
1962 refactor/add icons to upload modal

### PR DESCRIPTION
**Pull request by Ryoken**
#### What's this PR do?
Re-arranges the data upload modal to make a bit more sense and include icons.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/1962

#### Screenshots (if appropriate)
Screen Shot 2021-08-09 at 1 07 31 PM
Screen Shot 2021-08-09 at 1 07 40 PM
Screen Shot 2021-08-09 at 1 08 38 PM
Screen Shot 2021-08-09 at 1 08 43 PM

### CODE REVIEW

**Comment by Ryoken** [comment]
Updated based on comments on the ticket.

File: seed/static/seed/partials/data_upload_modal.html [hunk 1]
```diff
@@ -30,50 +30,113 @@ <h4 class="modal-title" ng-switch-when="16" translate>PM_METER_IMPORT_RESULTS</h
         </div>
     </div>
     <!-- Step 2: Choose upload type -->
```
Comment (macintoshpie) [inline seed/static/seed/partials/data_upload_modal.html:32]: Maybe we should update this comment now that this handles step `2` and step `13`

**Review by macintoshpie — COMMENTED** [review]
Tested all flows I could think of (and error cases) and everything looks great! One suggestion though is tweaking how we display things while a file is being uploaded. E.g. this is what a user sees right after selecting a file for upload. Maybe we should hide the entire tab section instead of just the buttons?

Screen Shot 2021-08-13 at 3 31 53 PM

**Comment by macintoshpie** [comment]
Oh yeah, also could you update the screenshots now that we've moved the cycles? Sorry I forgot to while testing 😬

**Comment by Ryoken** [comment]
Moved cycle:
Screen Shot 2021-08-16 at 12 56 38 PM

Cleaned up during upload:
Screen Shot 2021-08-16 at 1 36 18 PM

**Review by macintoshpie — APPROVED** [review]
😎  very nice

**Pull request merged**